standard/documentation-comments.md

```diff
@@ -368,10 +381,13 @@ where
 **Example:**
 
 ```csharp
-/// <permission cref="System.Security.PermissionSet">Everyone can
-/// access this method.</permission>
-public static void Test() {
-   // ...
+/// <permission cref="System.Security.PermissionSet">
+/// Everyone can
```

Comment:
Unwrap the two lines to one? (Side note: I'd never heard of the permission element before!)

Reply:
On the side note: This may be one we remove. The `permission` tag was to document any code access security (CAS) expectations. That's not recommended now. Thoughts?